app/tests/spec/views/sign_in_unblock.js

```diff
@@ -217,6 +217,19 @@ define(function (require, exports, module) {
         });
       });
 
+      describe('incorrect unblock_code', () => {
+        beforeEach(() => {
+          sinon.spy(view, 'showValidationError');
+          view.$('#unblock_code').val(UNBLOCK_CODE);
+
+          return view.submit();
+        });
+
+        it('displays a tooltip, does not call submit', () => {
+          assert.isTrue(view.showValidationError.calledWithExactly);
```

Comment:
It looks like `calledWithExactly` is not finished. Should the method be called with something?